compute/instances.go

```diff
@@ -1125,9 +1125,9 @@ var reservedInstanceCNSTags = map[string]struct{}{
 	CNSTagServices:   {},
 }
 
-// tagsExtractMeta() extracts all of the misc parameters from Tags and returns a
+// TagsExtractMeta() extracts all of the misc parameters from Tags and returns a
```

Comment:
comment on exported function TagsExtractMeta should be of the form "TagsExtractMeta ..."